components/clm/src/dyn_subgrid/dynTimeInfoMod.F90

```diff
@@ -144,7 +144,7 @@ end subroutine set_current_year
   ! ======================================================================
 
   !-----------------------------------------------------------------------
-  subroutine set_current_year_get_year(this)
+  subroutine set_current_year_get_year(this, offset)
```

Comment:
this looks fine to me, seeing how the original functionality of this routine is preserved, and should not impact existing calls to it, pinging @bishtgautam